README.md

```diff
@@ -28,32 +28,6 @@ A file based translation manager, which unlike other Lang managers don't need a
 
 `php artisan vendor:publish --provider="ctf0\Lingo\LingoServiceProvider"`
 
-- after installation, package will auto-add
-    + package routes to `routes/web.php`
-    + package assets compiling to `webpack.mix.js`
-    + `MIX_LINGO_FRAMEWORK=bulma` to `.env`
-
-- install dependencies
-
-```bash
-yarn add vue axios vue-ls vuemit vue-notif vue-clipboard2 vue-tippy vue-awesome
```

Comment:
the reason for keep the dependencies external is so you can use the same package anywhere else without the need to reinstall it

Reply:
No need to install any dependencies, only the dev need them